### ISSUE TITLE
V2VAM code is wrong ?

### ISSUE THREAD

**Issue by zhang-jing-zj** [DerrickXuNu/OpenCOOD#169]
''att = [] '' should be listed after  '' for xx in split_x:#split_x[0] [num_car, C, W, H] ''

class V2V_AttFusion(nn.Module):
    def __init__(self, feature_dim):
        super(V2V_AttFusion, self).__init__()
        

        self.cov_att = nn.Sequential(
                nn.Conv2d(in_channels=feature_dim, out_channels=feature_dim, kernel_size=3, padding=1),
                nn.BatchNorm2d(feature_dim,eps=1e-5, momentum=0.01, affine=True),
                nn.ReLU()
            )

        self.CCNet = CrissCrossAttention(feature_dim)
    def forward(self, x, record_len):

        split_x = self.regroup(x, record_len)  #x =[5, 64, 100, 352], record_len=[3,2]

        out = []
        att = []
        for xx in split_x:#split_x[0] [num_car, C, W, H]

            ''' CCNet: Criss-Cross Attention Module: attention for ego vehicle feature + cav feature '''

            ego_q, ego_k, ego_v = xx[0:1], xx[0:1], xx[0:1] 
            for i in range(len(xx[:,0,0,0])):
                att_vehicle = self.CCNet(ego_q, xx[i:i+1], xx[i:i+1])
                att.append(att_vehicle)

            pooling_max = torch.max(torch.cat(att, dim=0), dim=0, keepdim=True)[0]
            pooling_ave = torch.mean(torch.cat(att, dim=0), dim=0, keepdim=True)[0]

            fuse_fea = pooling_max + pooling_ave

            fuse_att = fuse_fea
            fuse_att = self.cov_att(fuse_att)

            out.append(fuse_att) #[[1, 64, 100, 352], [1, 64, 100, 352]]
            # torch.cuda.empty_cache()

        return torch.cat(out, dim=0) #[2, 64, 100, 352]

**Comment by zhang-jing-zj** [comment]
@jinlong17 So sorry to pardon you. Could you please tell this ?
